stage1/usr_from_coreos/coreos-common.mk

```diff
@@ -11,9 +11,13 @@ $(call setup-tmp-dir,CCN_TMPDIR)
 # systemd version in coreos image
 CCN_SYSTEMD_VERSION := v229
 # coreos image version
+ifeq ($(RKT_ACI_ARCH),aarch64)
+CCN_IMG_RELEASE := 1053.2.0
+else
 CCN_IMG_RELEASE := 1032.0.0
+endif
```

Comment:
PR up at https://github.com/coreos/rkt/pull/2821. @glevand: it shouldn't break arm64 related stuff, but please double-check.